.github/cloudbuild/pos-deploy-release.yaml

```diff
@@ -33,7 +33,6 @@ steps:
     gcloud container clusters get-credentials --zone $_MAIN_CLUSTER_ZONE $_MAIN_CLUSTER
     kubectl create namespace $_MAIN_NS-db
     kubectl create namespace $_MAIN_NS-inmemory
-    kubectl apply -f /workspace/namespace.yaml
```

Comment:
point of clarification - you're taking this out since the ns will already exist in this cluster?

Reply:
This was an error from a previous cleanup I did. Previously this whole step was broken into two:
- Create the `/workspace/namespace.yaml` file
- Then apply it on the next cloudbuild step
Then I realized I could just do it all at once like we see in line 34,35 above. After adding those two lines I forgot to delete this line. I noticed it when I was making changes to this PR.

Reply:
lol my eyes just glazed over. sounds good and makes sense to me!